### PR TITLE
Fix ::bot info not responding in dm

### PR DESCRIPTION
**Pull request by stepech**
Similar to #201 
Bot is unable to respond to `bot info` command in direct message since v1.0.0, this fixes it

### CODE REVIEW

File: src/HonzaBotner.Discord.Services/Commands/BotCommands.cs [hunk 1]
```diff
@@ -16,11 +16,15 @@ public class BotCommands : BaseCommandModule
     {
         private readonly ILogger<BotCommands> _logger;
         private readonly InfoOptions _infoOptions;
+        private readonly IGuildProvider _guildProvider;
 
-        public BotCommands(ILogger<BotCommands> logger, IOptions<InfoOptions> infoOptions)
+        public BotCommands(ILogger<BotCommands> logger,
+            IOptions<InfoOptions> infoOptions,
+            IGuildProvider guildProvider)
         {
```
Comment (tenhobi) [inline src/HonzaBotner.Discord.Services/Commands/BotCommands.cs]: Nit: put the first parameter on the next line.

Comment (tenhobi) [reply src/HonzaBotner.Discord.Services/Commands/BotCommands.cs]: And the trailing `)` on a separate line.

File: src/HonzaBotner.Discord.Services/Commands/BotCommands.cs [hunk 2]
```diff
@@ -34,7 +38,7 @@ public BotCommands(ILogger<BotCommands> logger, IOptions<InfoOptions> infoOption
             string status
         )
         {
-            if (type == ActivityType.Custom || type == ActivityType.Streaming)
+            if (type is ActivityType.Custom or ActivityType.Streaming)
```
Comment (tenhobi) [inline src/HonzaBotner.Discord.Services/Commands/BotCommands.cs:43]: Any reasoning why changing this? I thought you used the later version before and then changed it to the current one. Why change it again? :D

**Review by tenhobi — CHANGES_REQUESTED** [review]
Some nits, otherwise LGTM.

Also pls add a note to the changelog about this fix. ;)

File: src/HonzaBotner.Discord.Services/Commands/BotCommands.cs (hunk 2 above)
Comment (stepech) [reply src/HonzaBotner.Discord.Services/Commands/BotCommands.cs:43]: That was different file/commit/change where I was checking that something `is not` and it looked too confusing. I can discard this if you want..

Comment (stepech) [reply src/HonzaBotner.Discord.Services/Commands/BotCommands.cs:43]: viz [mega long ultra special link](https://github.com/fit-ctu-discord/honza-botner/pull/181/files#diff-ec6b98a1cc8b72715eaddeca8798d1c6d8e779b8679c6754ae534a2622738bf4R72)

**Review by tenhobi — APPROVED** [review]

**Pull request merged**